### PR TITLE
add some activities to handbook

### PR DESCRIPTION
**Pull request by Dedekind561**
### Common Content?

- [ ] Block/s

### Common Theme?

- [ ] Yes

Issue number: #issue-number

### Org Content?

Module | Sprint | Page Type | Block Type

## Checklist

- [ ] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [ ] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [ ] I have run my code to check it works
- [ ] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

**Review by SallyMcGrath — COMMENTED** [review]
LGTM

File: common-content/en/activities/tell-me-the-target/_index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+title="Tell me the target"
+description="You’ll have to state what the target outcome/output is when we run the code"
```
Comment (SallyMcGrath) [inline common-content/en/activities/tell-me-the-target/_index.md:3]: Hey, do we need this "you will have to" preamble?

Comment (Dedekind561) [reply common-content/en/activities/tell-me-the-target/_index.md:3]: Good catch - this is a hangover from getting messy ideas into a Google doc. I'll remove the preamble and make it all crisp and active

File: common-content/en/activities/in-your-own-words/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
++++
+title="In your own words"
+description="You’ll have to take an existing term and then explain it in their own words"
```
Comment (SallyMcGrath) [inline common-content/en/activities/in-your-own-words/_index.md:3]:
```suggestion
description="Take an existing term and then explain it in your own words"
```

File: common-content/en/activities/play-computer/_index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
++++
+title="Play computer"
+description="You’ll have to play the role of a computer by stepping through the execution of a computer program"
```
Comment (SallyMcGrath) [inline common-content/en/activities/play-computer/_index.md:3]:
```suggestion
description="Play the role of a computer by stepping through the execution of a computer program"
```

File: common-content/en/activities/tell-me-the-target/_index.md (hunk 1 above)
Comment (SallyMcGrath) [inline common-content/en/activities/tell-me-the-target/_index.md:3]:
```suggestion
description="State what the target outcome/output is when we run the code"
```

File: common-content/en/activities/where-is-the-code-running/_index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
++++
+title="Where is the code running?"
+description="You’ll have to identify where a piece of software is currently running. For example, is it being executed by a remote machine, by a local machine etc"
```
Comment (SallyMcGrath) [inline common-content/en/activities/where-is-the-code-running/_index.md:3]:
```suggestion
description="Identify where a piece of software is currently running. For example, is it being executed by a remote machine, by a local machine etc"
```

File: common-content/en/activities/which-line-is-it/_index.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
++++
+title="Which line is it?"
+description="You’ll have to specify the line in the source code where something is happening."
```
Comment (SallyMcGrath) [inline common-content/en/activities/which-line-is-it/_index.md:3]:
```suggestion
description="Specify the line in the source code where something is happening."
```

**Review by SallyMcGrath — APPROVED** [review]
rm the preamble

**Pull request merged**